test/server/server.js

```diff
@@ -0,0 +1,228 @@
+import { expect } from 'chai';
+
+import {
+  padSingleDigitWithZero,
+  formatDate
+} from '../../src/server/server';
+
+export const padSingleDigitWithZeroTest = () => {
```

Comment:
make sure to test the non-numeric string case

Reply:
Some of these tests are redundant, e.g. testing `padSingleDigitWithZero(2)` and `padSingleDigitWithZero(4)` is effectively the same thing.